base/src/main/proto/spine/options.proto

```diff
@@ -848,3 +842,37 @@ message EntityOption {
     //
     Visibility visibility = 2;
 }
+
+// Defines a marker for a given type or a set of types.
+//
+// The option may be used in two modes:
+//   - with the marker code generation;
+//   - without the marker code generation.
+//
+// When used with the code generation, language-specific markers are generated by the Protobuf
+// compiler. Otherwise, it is expected that the user creates such markers manually.
+//
+message IsOption {
+
+    // Enables the generation of marker interfaces.
+    //
+    // The generation is disabled by default.
+    bool generate = 1;
+
+    // The reference to Java interface.
+    //
+    // May be an FQN or a simple name. In the latter case, the interface should belong to the same
+    // Java package as the message class which implements this interface.
+    //
+    // The framework does not ensure the referenced type exists.
+    // If the generation is disabled, the Java type is used as-is. Otherwise, a corresponding Java
+    // interface is generated.
+    //
+    // A generated interface has no declared methods and extends `com.google.protobuf.Message`.
+    // The `.java` file is placed alongside with the code generated by the proto-to-java compiler.
+    // If specified with an FQN, the package of the generated type matches the FQN.
```

Comment:
This sentence seems to be a new paragraph, as it's about how to reference a type.